src/main/java/mate/academy/spring/security/CustomUserDetailsService.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.security;
+
+import static org.springframework.security.core.userdetails.User.withUsername;
```

Comment:
```suggestion
import static org.springframework.security.core.userdetails.User;
```
You should not import statics in this way. Then it is not clear which class the method belongs to